public/test/config.js

```diff
@@ -46,6 +46,11 @@ export default {
             'facia-tool-disable': false,
             'facia-tool-draft-content': true,
             'facia-tool-sparklines': false
-        }
+        },
+		baseUrls: {
+			mediaBaseUrl: 'https://media.code.dev-gutools.co.uk',
```

Comment:
I think this should be `https://media.test.dev-gutools.co.uk/`  as `.code` doesn't exist.